example/example-reader.py

```diff
@@ -0,0 +1,59 @@
+import copclib
+
+
+if __name__ == "__main__":
+
+    # Create a reader object
+    reader = copclib.FileReader("../build/test/data/autzen-classified.copc.laz")
```

Comment:
what are your thoughts on changing this to "copc" or something?

Reply:
I like it as `import copclib as copc` yep, will update to that